### PR TITLE
Documentation revision (./docs/*.md files)

### PR DESCRIPTION
**Pull request by Anirban166**
| | |
|:---|:---|
|**_Originating Project/Creator_**| @Anirban166 |
|**_Affected Component_**| N/A |
|**_Affected Architectures(s)_**| N/A |
|**_Related Issue(s)_**| -/- |
|**_Has Unit Tests (y/n)_**| n |
|**_Builds Without Errors (y/n)_**| y |
|**_Unit Tests Pass (y/n)_**| y |
|**_Documentation Included (y/n)_**| y |

---
## Change Description

Includes corrections for typographical errors and edits for better semantics and grammar for the content in the markdown files located in the `docs` directory.

While reviewing, I found a few paragraphs where the sentence structure is ambiguous. Among those, I fixed/rephrased the ones which I could understand well, but there are sections where I couldn't strictly determine what the original author(s) intended to say and so I left them unchanged.

## Rationale

To keep the documentation free of typos and to improve readability in general.

## Testing/Review Recommendations

I'm a bit concerned about this not being a small change (in size, across files) to review. I thought of going for a single PR instead of multiple ones since it might have been immaterial to do the latter, given that these changes involve no code (but are still subtly significant) and have patterns. Let me know if this needs to be segregated or decomposed into smaller PRs though, or if there is anything else I could help with regards to improving the documentation firsthand!

## Future Work

- #1251 perhaps

### CODE REVIEW

**Comment by LeStarch** [comment]
I'll be honest this is a big one to review.  Since it is all `.md` files (no code), I'll review it as is (no need to break it up).  It might take some time.

**Comment by Anirban166** [comment]
> I'll be honest this is a big one to review. Since it is all `.md` files (no code), I'll review it as is (no need to break it up). It might take some time.

I agree! It felt like a lot of scrolling after I saw the 'Files changed' section post submission of this PR. Thank you for taking the time to review this! (also @skovati!)

File: docs/Tutorials/GpsTutorial/Tutorial.md [hunk 1]
```diff
@@ -150,26 +148,26 @@ This file first imports all *Ai.xml files needed for each port type, imports our
...
 met. Port and component types are described in more detail in the aforementioned User Guide.
 
 Instantiating the GPS component, and connecting it with other components in the system is done at the system level,
-enabling the individual components to be reused in different applications. We will see this step later, after we design
-our Commands, Events and Telemetry. We will also implement the C++ code as well.
+enabling the individual components to be reused in different applications. We will see this step later after we design
+our Commands, Events, and Telemetry. We will also implement the C++ code as well.
 
-### Creating Commands.xml, Events.xml and Telemetry.xml Dictionaries
+### Creating Commands.xml, Events.xml, and Telemetry.xml Dictionaries
```
Comment (LeStarch) [inline docs/Tutorials/GpsTutorial/Tutorial.md:163]: Wooo  Oxford comma for the WIN!!!!

Comment (Anirban166) [reply docs/Tutorials/GpsTutorial/Tutorial.md:163]: Hell yeah, ten billion percent ftw!! :D

File: docs/Tutorials/MathComponent/md/Tutorial.md [hunk 2]
```diff
@@ -1052,7 +1051,7 @@ _(v1 + v2) f_.
 the event throttle.
 
 1. **Telemetry:**
-There two telemetry channels: one for reporting
+There are two telemetry channels - one for reporting
```
Comment (bocchino) [inline docs/Tutorials/MathComponent/md/Tutorial.md:1054]: A hyphen is not correct here. The original colon is correct. The correct alternative punctuation is an em-dash.

File: docs/Tutorials/MathComponent/md/Tutorial.md [hunk 3]
```diff
@@ -1039,7 +1038,7 @@ _(v1 + v2) f_.
       is updated by command.
       This event is **throttled** to a limit of three.
       That means that after the event is emitted three times
-      it will not be emitted any more, until the throttling
```
Comment (bocchino) [inline docs/Tutorials/MathComponent/md/Tutorial.md:1042]: "any more" is correct. See https://dictionary.cambridge.org/us/grammar/british-grammar/any-more-or-anymore. "anymore" seems informal/colloquial.

File: docs/Tutorials/MathComponent/md/Tutorial.md [hunk 4]
```diff
@@ -498,7 +497,7 @@ The implementation code does the following:
 1. Emit telemetry and events.
 
 1. Invoke the `mathOpOut` port to request that `MathReceiver`
-perform the operation.
+performs the operation.
```
Comment (bocchino) [inline docs/Tutorials/MathComponent/md/Tutorial.md:500]: "performs" is not correct here. "perform" is correct. The subjunctive mood indicates a request. See https://www.merriam-webster.com/words-at-play/getting-in-the-subjunctive-mood. (E.g., "I requested that he go to the store" not "I requested that he went to the store")

**Review by bocchino — COMMENTED** [review]
I reviewed the changes to the Math Tutorial. Some of the suggested changes are typo corrections. Thanks for those. A few suggested changes are not correct IMO. I added comments on those. The extra commas are a matter of personal preference or style. Personally I would not add most of them.

File: docs/Tutorials/MathComponent/md/Tutorial.md (hunk 4 above)
Comment (Anirban166) [reply docs/Tutorials/MathComponent/md/Tutorial.md]: Thanks for sharing information with links for the accepted usage! Appreciated. I corrected them or reverted the changes to the three sections you mentioned.

**Review by timcanham — APPROVED** [review]

**Pull request merged**